pkg/deploy/che_configmap.go

```diff
@@ -154,6 +159,8 @@ func GetConfigMapData(cr *orgv1.CheCluster) (cheEnv map[string]string) {
 		CheWebSocketEndpoint:                 wsprotocol + "://" + cheHost + "/api/websocket",
 		WebSocketEndpointMinor:               wsprotocol + "://" + cheHost + "/api/websocket-minor",
 		CheDebugServer:                       cheDebug,
+		CheDockerAlwaysPullImage:             cheDockerAlwaysPullImage,
+		CheWorkspaceSidecarImagePullPolicy:   cheWorkspaceSidecarImagePullPolicy,
```

Comment:
> CheWorkspaceSidecarImagePullPolicy
> CheDockerAlwaysPullImage

IMO we shouldn't override it by default with a constant, only when airgap is enabled, and possibly with a different value than the default one provided by `che.properties`

I'm not even sure that we have to override them for airgap

Reply:
yeah we might be OK here since the assumption is that the newer images will also be inside the airgap environment. 

As long as the image r.r.io/crw/server:2.0 -> myquay.bigcorp.com/crw-stuff/crw-server:2.0 and newer ones are managed under the same tag... then no policy change should be needed.

I mean pullpolicy of Always pull might annoy teh devops team at bigcorp as it's extra network traffic on every startup but that'd be up to them to decide when installing Che/CRW in teh first place... right?

Reply:
yeah. So let not override those at all for now